monitor/grafana/zeebe.json

```diff
@@ -253,10 +253,10 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(zeebe_flow_node_instance_counter{namespace=~\"$namespace\", pod=~\"$pod\"}[1m])) by (partition, type)",
+              "expr": "sum(rate(zeebe_element_instance_events_total{namespace=~\"$namespace\", pod=~\"$pod\", type!=\"PROCESS\", action!=\"activated\"}[1m])) by (partition, action)",
```

Comment:
I just thought about something - do we also count terminated here? I just saw `activated`/`completed`, but of course it may have been because none of them were terminated. Did you verify? I'm asking because I'm not sure if `action !="activated"` is enough.

Reply:
We count `terminated` I verified it and this is how it was before we want to show both.

Reply:
@npepinpe

Reply:
Ok, so this count will be exactly completed + terminated? :+1:

Reply:
yes